spec/controllers/your_details_controller_spec.rb

```diff
@@ -54,33 +65,24 @@ module Wpcc
       context 'success' do
         after { Timecop.return }
 
-        let(:your_details_form) { Wpcc::YourDetailsForm.new(nil_session) }
-        let(:nil_session) do
-          {
-            'age' => nil,
-            'gender' => nil,
-            'salary' => nil,
-            'salary_frequency' => nil,
-            'contribution_preference' => nil
-          }
-        end
+        let(:your_details_form) { Wpcc::YourDetailsForm.new(some_nil_session) }
 
         it 'stores the form input in a session for 30 minutes' do
```

Comment:
Do you think this test is actually testing two different things?
Setting the session
AND
the expiration of the session?

Could we move the first list of expectations into a seperate test. wdyt?